packages/Parallel/WorkerRunner.php

```diff
@@ -30,11 +32,15 @@ public function __construct(
         private readonly ParametersMerger $parametersMerger,
         private readonly CurrentFileProvider $currentFileProvider,
         private readonly PhpFileProcessor $phpFileProcessor,
+        private readonly NodeScopeResolver $nodeScopeResolver,
+        private readonly DynamicSourceLocatorDecorator $dynamicSourceLocatorDecorator,
     ) {
     }
 
     public function run(Encoder $encoder, Decoder $decoder, Configuration $configuration): void
     {
+        $this->dynamicSourceLocatorDecorator->addPaths($configuration->getPaths());
```

Comment:
The 1st missing part was paths here. That's how we can tell Rector and PHPStan to use static reflection on specific directories